.github/workflows/docker_build.yml

```
@@ -0,0 +1,31 @@
+name: Docker Build Verification
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)